### DISCUSSION TITLE
Many vendor files updated after running 'go mod tidy, 'go mod vendor'

### DISCUSSION THREAD

**Discussion by mghildiy** [grafana/tempo#2306] [General]
In my work on one of the issue, I was unable to run related test. After some google search, I ran commands:

- go mod tidy
- go mod vendor

So issues were gone, and I could run the test.

But these commands also changed many files(all related to vendor) which are not part of my work.

Is it expected?

**Comment by zalegrala** [comment]
This can happen in some cases due to local cache differences among machines.  In your PR, are you bringing in new dependencies?

**Reply by mghildiy** [reply]
I have added imports:

"github.com/go-kit/log/level"
"github.com/grafana/tempo/pkg/util/log"

But these are already present in many other source files, so no new dependencies.

**Reply by zalegrala** [reply]
Do you have a link to the PR?  I'm guessing that perhaps they got updated because your local cache didn't contain the dependencies.

**Reply by mghildiy** [reply]
Here it is:

https://github.com/grafana/tempo/pull/2333

**Reply by zalegrala** [reply]
Perhaps just a rebase on `main` will resolve the issues you're running into.
